src/action/nav.js

```diff
@@ -31,6 +31,10 @@ class NavAction {
     this._store.route = 'NewAddress';
   }
 
+  goWait() {
```

Comment:
I guess it doesn't matter if it's really temporary, but I'd go with a more descriptive name like `waitForNetworkLoad`. Unless we plan on having 1 `Wait` screen for multiple types of waiting?

Reply:
Yes. I also added the wait screen for the lightning payment. That takes a few seconds on testnet